core/domain/stats_services.py

```diff
@@ -40,6 +40,31 @@ def get_top_unresolved_answers_for_default_rule(exploration_id, state_name):
     }
 
 
+def get_exp_wise_unresolved_answers_count_for_default_rule(exp_ids):
+    exploration_objects = exp_services.get_multiple_explorations_by_id(exp_ids)
```

Comment:
(Optional) Elsewhere in the codebase, we generally use `explorations` to refer to exploration domain objects, so you can consider renaming this here.